### DISCUSSION TITLE
Prettify GrahpQL

### DISCUSSION THREAD

**Discussion by azzamsa** [usebruno/bruno#342] [Ideas]
Hi.

It would be great if Bruno had the "Prettify GraphQL" or "Prettify JSON" or "Prettify < anything>" button.

![image](https://github.com/usebruno/bruno/assets/17734314/8a4cdafe-e640-47a5-9d59-cd47e5cec0ce)

**Comment by j0k3r** [comment]
It'll be available in the next release: https://github.com/usebruno/bruno/pull/526

**Comment by helloanoop** [comment]
This is now available in `v1.10.0`
Clicking on the wand icon will prettify the graphql query

![image](https://github.com/usebruno/bruno/assets/9350733/8368dbcf-51c3-4fbd-98ef-1fc02bdd71b3)
